src/chainparams.cpp

```diff
@@ -338,28 +338,109 @@ class CRegTestParams : public CChainParams {
     }
 };
 
+/**
+ * Custom params for testing.
+ */
+class CCustomParams : public CChainParams {
+
+    void UpdateFromArgs(ArgsManager& args)
+    {
+        consensus.fPowAllowMinDifficultyBlocks = args.GetBoolArg("-con_fpowallowmindifficultyblocks", true);
```

Comment:
When it is possible to derive from the RegTestPrams class, you might want to put the current value (i.e. `consensus.fPowAllowMinDifficultyBlocks` instead of the hardcoded value here?